apps/discovery_streams/test/unit/presence_test.exs

```diff
@@ -0,0 +1,56 @@
+defmodule DiscoveryStreamsWeb.PresenceTest do
+  use DiscoveryStreamsWeb.ChannelCase
+  use Prometheus.Metric
+  use Placebo
+
+  # import Checkov
```

Comment:
Should this guy still be completely commented out?

Reply:
Yeah, I guess, I dunno